Example/Info.plist

```diff
@@ -2,6 +2,11 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
```

Comment:
Setting this will allow `HTTP` connections? Is there a reason this is needed?

Reply:
No. It will allow only when NSExceptionAllowsInsecureHTTPLoads is added in NSAppTransportSecurity

Reply:
Why do we need it? The backend doesn't have proper https / cert?

@corycaywood this is a workaround that allows all connections, including insecure ones.

Reply:
I thought the staging is not gonna have one. Looks it is having valid one. Will update the plist.